package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "restaurant-page-JS",
+  "version": "1.0.0",
+  "description": "This is a project from [the Odin project](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page) for implementation of webpack and continued DOM manipulation in Javascript",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] Great job using `webpack` on your project and adding a script for it. However, it's best practice to create at least 2 scripts, for development and production stages. In each script, we should inform in which mode Webpack will run. We will have something like this:
```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
  },
```

- [ ] Remember that after adding those scripts we also need to inform about them at the README file, at the usage (or similar) section. This is important because will aid users to use your application.

**OPTIONAL**

- [ ] Add the `webpack-dev-server` dependency to be used by you in development stages or by others when trying your app. You can create a script for it and add the `--open` flag, this way you won't need to open the index.html file because it will happen automatically.

```javascript
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },
```